tests/conf/i18n/translationresolver.js

```diff
@@ -0,0 +1,77 @@
+const TranslationResolver = require('../../../conf/i18n/translationresolver');
+const Translator = require('../../../conf/i18n/translator');
+const TranslationPlaceholder = require('../../../conf/i18n/models/translationplaceholder');
+
+async function createTranslationResolver () {
+  const translator = await createTranslator();
+  const passThroughRuntimeGenerator = translationResult => translationResult;
+  return new TranslationResolver(translator, passThroughRuntimeGenerator);
+}
+
+async function createTranslator () {
+  const locale = 'fr';
+  return Translator.create(locale, [], { [locale]: { translation: {
+    'Hello': 'Bonjour',
+    'result': 'résultat',
+    'result_plural': 'résultats',
+    'object_noun': 'objet',
+    'object_noun_plural': 'objets',
+    'mail_noun': 'courrier'
+  } } });
+}
+
+describe('TranslationResolver usage', () => {
```

Comment:
can we replace "usage" with something more descriptive (just slightly more descriptive don't have to go overboard) also for the other describe blocks

Reply:
sure